src/Time/StepChoosers/CFL.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <pup.h>
+
+#include "Domain/MinimumGridSpacing.hpp"
+#include "Options/Options.hpp"
```

Comment:
Is there no reason to use your trick to try and avoid some options related includes here?

Reply:
I don't understand.

Reply:
Disregard :) I was confusing the `ParseOptions` and the `Options` header